build/commands/scripts/updatePatches.js

```diff
@@ -17,26 +17,19 @@ function loadChromiumPathFilter(filePath) {
 
   const prefixes = [];
   const suffixes = [];
-  const exactMatches = new Set();
 
-  // This should be revisited in the future once `path.matchesGlob` is stable
-  // and available in node to use, as this current implementation is a bit
-  // naive.
+  // This can be revisited once `path.matchesGlob` is available on stable node.
```

Comment:
seems this is in lts now? https://nairihar.medium.com/what-is-new-in-node-js-v22-5-8899620ddebf

Reply:
I push this as a separate PR.